internal/cli/atlas/deployments/setup.go

```diff
@@ -730,10 +734,62 @@ func (opts *SetupOpts) Run(ctx context.Context) error {
 	return opts.runConnectWith(cs)
 }
 
+func (opts *SetupOpts) RunAtlas(ctx context.Context) error {
```

Comment:
I don't see the `_test.go` file. Do we plan to add it in a follow up PR?

Reply:
yes, we only have e2e so far let's add for both RunAtlas and Local as a follow-up